creator-node/src/serviceRegistry.js

```diff
@@ -290,7 +290,9 @@ class ServiceRegistry {
       identityServiceConfig: identityService ? AudiusLibs.configIdentityService(identityService) : undefined,
       isDebug: config.get('creatorNodeIsDebug'),
       isServer: true,
-      enableUserReplicaSetManagerContract: true
+      enableUserReplicaSetManagerContract: true,
+      // TODO: need to wire up optimizely for this flag
+      snapbackReconfigEnabled: config.get('snapbackReconfigEnabled')
```

Comment:
i'm not too familiar with optimizely but don't think this would suffice? we may actually need to wire it up to cnode directly. maybe you can check with piazza or other

Reply:
discussed offline - check identity

Reply:
not consuming this flag via optimizely, just config